packages/atomic/src/components/commerce/atomic-commerce-breadbox/atomic-commerce-breadbox.ts

```diff
@@ -40,6 +40,7 @@ import {
   defaultCurrencyFormatter,
   defaultNumberFormatter,
 } from '../../common/formats/format-common';
+import {ValidatePropsController} from '../../common/validate-props-controller/validate-props-controller';
```

Comment:
- [ ] Create Jira to use validate props controller in every initializable component that was migrated to Lit and that has props to validate.